requirements.in

```diff
@@ -41,7 +41,7 @@ django-hijack-admin==2.1.7
 -e git+https://github.com/mitodl/mit-moira.git#egg=mit-moira==0.0.4
 
 # Django-Elastic-Transcoder
--e git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder
+-e git+https://github.com/mitodl/django-elastic-transcoder.git@8d97d5760eb687ac7fb879a9c1ff1c02e5ae6f6f#egg=django-elastic-transcoder
```

Comment:
Change to:
`-e git+https://github.com/mitodl/django-elastic-transcoder.git#egg=django-elastic-transcoder==0.9.7`